template/test/unit/specs/Hello.spec.js

```diff
@@ -5,7 +5,7 @@ describe('Hello.vue', () => {
   it('should render correct contents', () => {
     const vm = new Vue({
       el: document.createElement('div'),
-      render: (h) => h(Hello){{#if_eq lintConfig "airbnb"}},{{/if_eq}}
+      render: {{#if_eq lintConfig "airbnb"}}h => h(Hello),{{/if_eq}}{{#unless_eq lintConfig "airbnb"}}(h) => h(Hello){{/unless_eq}}
```

Comment:
This can be shortened to:

```
render: h => h(Hello){{#if_eq lintConfig "airbnb"}},{{/if_eq}}
```

Because parentheses-less single argument arrow functions are acceptable in both linting configs.